src/plugins/plot/chart/MctChart.vue

```diff
@@ -196,6 +197,7 @@ export default {
             this.listenTo(series, 'change:alarmMarkers', this.changeAlarmMarkers, this);
             this.listenTo(series, 'change:limitLines', this.changeLimitLines, this);
             this.listenTo(series, 'change:yAxisId', this.resetAxisAndRedraw, this);
+            // TODO: Which other changes is the listener below reacting to?
```

Comment:
Ah, event based listeners 😀 It may help in the future to have more specific names for the events. Like `changeSeries` instead of just `change`. Or at least it would help searching for the event name in the code base.